src/server/pegasus_server_impl.cpp

```diff
@@ -274,6 +281,12 @@ void pegasus_server_impl::on_get(get_rpc rpc)
     resp.partition_index = _gpid.get_partition_index();
     resp.server = _primary_address;
 
+    if (!_read_size_throttling_controller->get_token(FLAGS_read_throttling_by_size_request_count)) {
```

Comment:
exactly should getToken(rpc.size)

Reply:
Actually, it has nothing to do with any size, it only checks if the token basket is empty. If it's empty, then reject all requests

Reply:
You have define the default value in get_token, no need pass the FLAGS_read_throttling_by_size_request_count

Reply:
it can be remotely changed if necessary

Reply:
You just get current the avaliable token, so which case need change? or why change